ci/check-documentation.py

```diff
@@ -114,8 +114,25 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
-        # check for existence of links to README.md files of models and demos
+        # check <omz_dir> link validity
```

Comment:
```suggestion
        # check <omz_dir> reference validity
```

Your function is called `omz_references`, so you should call them references elsewhere, as well.